colossalai/auto_parallel/solver/cost_graph.py

```diff
@@ -83,7 +98,7 @@ def merge_node(self, src_node, dst_node):
         # build merge_map
         merge_map = {}
         for src_index, strategy in enumerate(src_node.strategies_vector):
-            min_cost = math.inf
+            min_cost = INFINITY_COST + 1
```

Comment:
Why plus one?

Reply:
The highest reshard cost is INFINITY_COST, so it could make sure at least one strategy could be picked.